src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp

```diff
@@ -0,0 +1,665 @@
+    }
+
+    size_t min_src_size = jep.dst_size;
+    for (size_t i = 0; i < jep.inputs_number; i++) {
+        if (jep.src_size[i] != 1)
+            min_src_size = std::min(min_src_size, jep.src_size[i]);
+    }
+    if (jep_.oc_size > 1)
+        min_src_size = std::min(min_src_size, jep_.oc_size);
+
+    if (min_src_size != jep.dst_size) {
+        bool is_valid_configuration = true;
+        if (jep.dst_size % min_src_size != 0)
+            is_valid_configuration = false;
+
+        for (size_t i = 0; i < jep.inputs_number; i++) {
+            if (jep.src_size[i] != 1 && jep.src_size[i] != min_src_size && jep.src_size[i] != jep.dst_size)
+                is_valid_configuration = false;
+        }
+
+        if (jep.oc_size > 1 && jep.oc_size != min_src_size && jep.oc_size != jep.dst_size)
+            is_valid_configuration = false;
+
+        if (!is_valid_configuration)
+            IE_THROW() << "Eltwise jitter has invalid configuration for Eltwise node";
+
+        L(unroll_loop_label);
+        {
+            const size_t loop_step = min_src_size;
+            const size_t vec_step = cpu_isa_traits<isa>::vlen / exec_prc.size();
+
+            cmp(reg_work_amount, loop_step);
+            b(LO, unroll_loop_end_label);
+
+            for (size_t j = 0; j < min_src_size / vec_step; j++) {
+                for (size_t i = 0; i < jep.inputs_number; i++) {
+                    if (jep.src_size[i] != 1) {
+                        load_vector(get_vmm_reg(i), get_src_reg(i), jep.src_prc[i], exec_prc, false, j * vec_step * jep.src_prc[i].size());
+                    }
+                }
+
+                compute_eltwise_op();
+
+                apply_post_ops();
+
+                store_vector(reg_dst, vmm_dst, exec_prc, jep.dst_prc, j * vec_step * jep.dst_prc.size());
+            }
+
+            size_t tail_start = min_src_size - min_src_size % vec_step;
+            for (size_t j = tail_start; j < min_src_size; j++) {
+                for (size_t i = 0; i < jep.inputs_number; i++) {
+                    if (jep.src_size[i] != 1) {
+                        load_scalar(get_scl_reg(i), get_src_reg(i), jep.src_prc[i], exec_prc, j * jep.src_prc[i].size());
+                    }
+                }
+
+                compute_eltwise_op();
+
+                apply_post_ops();
+
+                SReg sc_dst_reg{vmm_dst.getIdx()};
+                store_scalar(reg_dst, sc_dst_reg, exec_prc, jep.dst_prc, j * jep.dst_prc.size());
+            }
+
+            for (size_t i = 0; i < jep.inputs_number; i++)
+                if (jep.src_size[i] == jep.dst_size)
+                    add(get_src_reg(i), get_src_reg(i), jep.src_prc[i].size() * loop_step);
+
+            add(reg_dst, reg_dst, jep.dst_prc.size() * loop_step);
+            sub(reg_work_amount, reg_work_amount, loop_step);
+            if (jep_.oc_size > 1 && jep_.oc_size != min_src_size)
+                IE_THROW(NotImplemented) << "jit_uni_eltwise_generic<isa>::generate: reg_oc_off";
```

Comment:
Why it is not implemented? What is technical reason?

Reply:
was not used before, done